### PR TITLE
h3-python bindings

### PR DESCRIPTION
**Pull request by geoHeil**
Add python bindings from https://github.com/uber/h3-py to https://github.com/conda-forge/h3-feedstock.

How can I overwrite https://github.com/uber/h3-py/blob/master/setup.py to use the conda provided H3 https://github.com/conda-forge/h3-feedstock

@conda-forge/help-python-c looking forward to your feedback.

### CODE REVIEW

File: recipes/h3-py/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  sha256: a292a5a63bcf535f0137141019f3defd21ad2fbd258dc8de587f921cf2c5b41d
+
+build:
+  number: 0
+  skip: True  # [win]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - h3 # TODO: how can I overwrite https://github.com/uber/h3-py/blob/master/setup.py to use the conda provided H3 https://github.com/conda-forge/h3-feedstock
```
Comment (geoHeil) [inline recipes/h3-py/meta.yaml]: How can I overwrite to use https://github.com/conda-forge/h3-feedstock?

Comment (isuruf) [reply recipes/h3-py/meta.yaml]: You can patch the file. See https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#patches

Comment (isuruf) [reply recipes/h3-py/meta.yaml]: Btw, `h3` should be in `host`. (`build` should be empty for this package)

Comment (geoHeil) [inline recipes/h3-py/meta.yaml]: Where do I need to specify the h3 version?

Comment (geoHeil) [reply recipes/h3-py/meta.yaml]: Moved. Regarding patching:
the `.install.sh` file (https://github.com/uber/h3-py/blob/master/.install.sh) only contains the native code part, not python. So I assume this is already completely handled by the new conda-force h3 package.

I.e. I only need to patch https://github.com/uber/h3-py/blob/master/setup.py is this correct?

Comment (geoHeil) [reply recipes/h3-py/meta.yaml]: But patching fails with:
```
ile "/opt/conda/lib/python3.7/site-packages/conda_build/source.py", line 518, in _guess_patch_strip_level
    histo = {i: 0 for i in range(maxlevel + 1)}
TypeError: unsupported operand type(s) for +: 'NoneType' and 'int'
```

**Comment by benhuff** [comment]
@geoHeil i finally got a chance to take a look at the python bindings for h3-py. i am going to make a new pr for it and i will tag you as a maintainer.

**Pull request closed without merging**